packages/array-product/README.md

```diff
@@ -0,0 +1,11 @@
+## just-array-product
+
+Part of a [library](../../../../) of zero-dependency npm modules that do just do one thing.
+Guilt-free utilities for every occasion.
+
+```
+import product from 'just-array-product;
+
+product([[1, 2], ['a', 'b']); // [[1, 'a'], [1, 'b'], [2, 'a'], [2, 'b']]
```

Comment:
Extra examples in README please: 
e.g. `product([])` and `product([[1, 2], ['a', 'b', 'c'])`